posthog/models/test/__snapshots__/test_async_deletion_model.ambr

```diff
@@ -3,127 +3,127 @@
   '''
   
   ALTER TABLE sharded_events ON CLUSTER 'posthog'
-  DELETE
+  UPDATE is_deleted = 2
```

Comment:
This is a bit strange, but it is like this because for `ALTER` statements we are setting `replacing_all_numbers` to `True` when calling `assertQueryMatchesSnapshot`.

I did not want to introduce disruption, so I left it like this, but can be discussed of course.